test/service/test_app_logging.py

```diff
@@ -21,62 +25,114 @@
+                                INFO,
+                                "Authenticated request as OAuth2(access_token='foo_token')"
+                                if authenticated else
+                                'Did not authenticate request.'
+                            ),
+                            (
+                                level,
+                                'Returning 200 response. To log headers and body, set AZUL_DEBUG to 1.'
+                                if level == INFO else
+                                'Returning 200 response with headers {"Access-Control-Allow-Origin": '
+                                '"*", "Access-Control-Allow-Headers": '
+                                '"Authorization,Content-Type,X-Amz-Date,X-Amz-Security-Token,X-Api-Key", '
+                                f'"Content-Security-Policy": "default-src {sq("self")}", '
+                                '"Referrer-Policy": "strict-origin-when-cross-origin", '
+                                '"Strict-Transport-Security": "max-age=63072000; includeSubDomains; preload", '
+                                '"X-Content-Type-Options": "nosniff", '
+                                '"X-Frame-Options": "DENY", '
+                                '"X-XSS-Protection": "1; mode=block", '
+                                '"Cache-Control": "no-store"}. '
+                                'See next line for the first 1024 characters of the body.\n'
+                                '{"pagination": {"count": 1, "total": 1, "size": 10, "next": null, "previous":'
+                                ' null, "pages": 1, "sort": "projectTitle", "order": "asc"}, "termFacets": '
+                                '{"organ": {"terms": [{"term": "pancreas", "count": 1}], "total": 1, "type": '
+                                '"terms"}, "sampleEntityType": {"terms": [{"term": "specimens", "count": 1}], '
+                                '"total": 1, "type": "terms"}, "dataUseRestriction": {"terms": [{"term": null, '
+                                '"count": 1}], "total": 1, "type": "terms"}, "project": {"terms": [{"term": '
+                                '"Single of human pancreas", "count": 1, "projectId": '
+                                '["e8642221-4c2c-4fd7-b926-a68bce363c88"]}], "total": 1, "type": "terms"}, '
+                                '"sampleDisease": {"terms": [{"term": "normal", "count": 1}], "total": 1, "type": '
+                                '"terms"}, "nucleicAcidSource": {"terms": [{"term": "single cell", "count": 1}], '
+                                '"total": 1, "type": "terms"}, "assayType": {"terms": [{"term": null, "count": 1}], '
+                                '"total": 0, "type": "terms"}, "instrumentManufacturerModel": {"terms": [{"term": '
+                                '"Illumina NextSeq 500", "count": 1}], "total": 1, "type": "terms"}, "institution": '
+                                '{"terms": [{"term": "Farmers Tru',
```

Comment:
Please find a way to make this less ugly. For example, you could specify headers and body as JSON literals and use json.dumps() to convert them to a string, and taking only the first 1024 charterers of the body.